### PR TITLE
update to gcc 6.3.0 and binutils 2.28

### PR DESCRIPTION
**Pull request by ladislas**

### CODE REVIEW

**Comment by ladislas** [comment]
@osx-cross/team should be easy to merge, just used the latest versions

File: avr-gcc.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
 class AvrGcc < Formula
   homepage "https://www.gnu.org/software/gcc/gcc.html"
-  url "ftp://gcc.gnu.org/pub/gcc/releases/gcc-6.2.0/gcc-6.2.0.tar.bz2"
-  mirror "https://ftpmirror.gnu.org/gcc/gcc-6.2.0/gcc-6.2.0.tar.bz2"
-  sha256 "9944589fc722d3e66308c0ce5257788ebd7872982a718aa2516123940671b7c5"
+  url "ftp://gcc.gnu.org/pub/gcc/releases/gcc-6.3.0/gcc-6.3.0.tar.gz"
```
Comment (larsimmisch) [inline avr-gcc.rb]: Any particular reason why this is .gz instead of bz2? (not that it matters)

**Review by larsimmisch — APPROVED** [review]
Fine with me

File: avr-gcc.rb (hunk 1 above)
Comment (ladislas) [reply avr-gcc.rb]: no I just saw that, let me change it, the file is smaller in bz2, that's always better :)

**Comment by ladislas** [comment]
I just changed everything to `.bz2`, can I merge?

**Comment by ladislas** [comment]
yeah!

**Pull request merged**